onnx/reference/reference_evaluator.py

```diff
@@ -542,17 +542,27 @@ def _load_impl(  # noqa: PLR0911
             f"is unknown, known functions: {sorted(self.functions_)}."
         )
 
-    def run(self, output_names, feed_inputs: Dict[str, Any], attributes: Optional[Dict[str, Any]] = None):  # type: ignore
+    def run(
```

Comment:
Would it make sense to create a new function that will return intermediates, so the behavior (and return type) is clear?

Reply:
I thought of it but it would not be called in any other place. Since the change is quite small, I kept it in the same function.